pyflow/blocks/widgets/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Pyflow an open-source tool for modular visual programing in python
+# Copyright (C) 2021 Mathïs FEDERICO <https://www.gnu.org/licenses/>
+
+""" Module for the OCB Blocks Widgets. """
+
+from pyflow.blocks.widgets.blocksplitter import OCBSplitter
```

Comment:
Aren't those removed ?

Reply:
I am not sure, what do you want to do?

Reply:
No sorry I've mistaken this for Slider xD